language-design.md

```diff
@@ -116,6 +276,12 @@ Briefly describe how you split up the work for this assignment.
 
 **Response**
 
+We read the articles separately. We sat next to each other and talked through each of the questions and simultaneously developed an outline for each of the questions. We split up the final write-up. However, we did an unfair initial split and adjusted for it as we did the write-up.
```

Comment:
Your workflow was extensible enough to cope with the differences between expectation and reality :)